binzx/otomi

```diff
@@ -231,12 +232,7 @@ if [ -n "$in_core" ]; then
   stack_volume="-v $stack_dir:$stack_dir"
   tmp_volume="-v /tmp:/tmp"
 
-  # Use development distribution if exists
-  if [ -f "${stack_dir}/dist/otomi.js" ]; then
-    executable="node --experimental-specifier-resolution=node ${stack_dir}/dist/otomi.js --"
-  else
-    executable="node --no-warnings --experimental-specifier-resolution=node --loader ts-node/esm ${stack_dir}/src/otomi.ts --"
-  fi
+  executable="node --no-warnings --experimental-specifier-resolution=node --loader ts-node/esm ${stack_dir}/src/otomi.ts --"
```

Comment:
Duplicate of https://github.com/redkubes/otomi-core/pull/593/files#diff-2d567601f6715e841da329cb4f20186f3aaeda8fd838c89278dc9ac2ad287495R264

Reply:
Yes, needed to remove it for testing purposes. Can't hurt to leave it there right?